pkg/cache/v3/delta_test.go

```diff
@@ -359,6 +359,7 @@ func TestSnapshotCacheDeltaWatchWithForceEDSOfRelevantEndpoints(t *testing.T) {
 		vMap := out.GetNextVersionMap()
 		versionMap[testTypes[1]] = vMap
 	case out := <-watches[testTypes[0]]:
+		assert.Len(t, out.(*cache.RawDeltaResponse).Resources, 1)
```

Comment:
this is a nitpick but why do we need a for loop, can't we just do something like:

```suggestion
		assert.Len(t, out.(*cache.RawDeltaResponse).Resources, 1)
		assert.Equal(t, out.(*cache.RawDeltaResponse).Resources[0], testEndpoint)
```